### ISSUE TITLE
No error for generic Protocol with ParamSpec

### ISSUE THREAD

**Issue by 0e4ef622** [microsoft/pyright#5151]
**Describe the bug**
```py
# pyright: strict
from typing import ParamSpec, Protocol, TypeVar

P = ParamSpec("P")
T = TypeVar("T", covariant=True)

class Pr(Protocol[P, T]):
    def f(self, *args: P.args, **kwargs: P.kwargs) -> T | None:
        ...


class A(Pr[P, T]):
    # missing params
    def f(self) -> T | None:
        pass
```

**To Reproduce**
Run pyright on the code block.

**Expected behavior**
Should report an incompatible method override in strict mode.

**VS Code extension or command-line**
pyright command line version 1.1.309

**Comment by erictraut** [comment]
This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.310, which I just published. It will also be included in a future release of pylance.
